File: api/src/main/java/com/makingiants/today/api/repository/history/pojo/Event.java
package com.makingiants.today.api.repository.history.pojo;

import com.google.gson.annotations.SerializedName;

public class Event {
  String title;
  String date;
  @SerializedName("url") String imageUrl;

  public Event(String title, String date, String imageUrl) {
    this.title = title;
    this.date = date;
    this.imageUrl = imageUrl;
  }

  public String getTitle() {
    return title;
  }

  public String getDate() {
    return date;
  }

  public String getImageUrl() {
    return imageUrl;
  }
}

File: api/src/main/kotlin/com/makingiants/today/api/repository/history/pojo/Event.kt
package com.makingiants.today.api.repository.history.pojo

import com.google.gson.annotations.SerializedName

class Event(var title: String,
            var date: String,
            var thumb: Thumb)

class Thumb(@SerializedName("w") var weight: Int,
            @SerializedName("h") var height: Int,
            val src: String)